src/http-driver.js

```diff
@@ -128,7 +128,7 @@ function makeHTTPDriver({eager = false} = {eager: false}) {
       .map(request => {
         const reqOptions = normalizeRequestOptions(request)
         let response$ = createResponse$(reqOptions)
-        if (eager || reqOptions.eager) {
+        if (reqOptions.hasOwnProperty(`eager`) ? reqOptions.eager : eager) {
```

Comment:
Can we replace `hasOwnProperty` with `typeof reqOptions.eager === 'boolean'`? I suspect the latter is a tiny bit faster.